config/application.yml.default

```diff
@@ -445,6 +438,7 @@ production:
   kantara_2fa_phone_restricted: false
   kantara_2fa_phone_existing_user_restriction: false
   kantara_restriction_enforcement_date: '2022-07-19'
+  lexisnexis_threatmetrix_mock_enabled: false
```

Comment:
line 176 has mock_enabled: true

Reply:
Yeah, the goal here is to _enable_ it locally by default but disable it by default in production